runFlask.sh

```diff
@@ -1,14 +1,9 @@
 #!/bin/bash
 
 VAG_COMMON_DIR="/vagrant"
-HOME="/home/vagrant"
 LOG_DIR="$VAG_COMMON_DIR/log"
 FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
-FLASK_DIR=$HOME/FlaskApp
+FLASK_LOG="$LOG_DIR/flask-out.log"
 
-cd $FLASK_DIR
-if [ -e venv/bin/activate ]; then
-    source venv/bin/activate
-    export FLASK_APP=$FLASK_APP_PY 
-    flask run --host=0.0.0.0 >> /dev/null 2>&1 &
-fi
+export FLASK_APP=$FLASK_APP_PY 
+flask run --host=0.0.0.0 >> $FLASK_LOG 2>&1 &
```

Comment:
new line at the end of file

Reply:
I could swear I put it there. Will bee added next PR.
BTW, Indeed all code tested, and sudo never prompted for password.